js/scale.js

```diff
@@ -0,0 +1,29 @@
+const scale = document.querySelector('.img-upload__scale ');
+const scaleSmaller = scale.querySelector('.scale__control--smaller');
+const scaleBigger = scale.querySelector('.scale__control--bigger');
+const scaleValue = scale.querySelector('.scale__control--value');
+const picture = document.querySelector('.img-upload__preview');
+
+
+const getBiggerScale = () => {
+  if (scaleValue.value !== '100%') {
+    scaleValue.value = `${Number(scaleValue.value.slice(0, scaleValue.value.length - 1)) + 25}%`;
```

Comment:
```suggestion
    scaleValue.value = `${Number.parseFloat(scaleValue.value) + 25}%`;
```

[parseFloat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat) отлично здесь подойдет)

Исправь, пожалуйста, во **всех функциях**